core/idl/wallet/cosmos/messages.djinni

```diff
@@ -172,6 +172,10 @@ CosmosLikeMessage = interface +c {
         # @return string the log of the message execution (useful when getSuccess() == false)
         const getLog(): string;
 
+        # Get message index in the tx
+        # @return string the 0-based index in the tx or "fees" for the fees in the tx
+        const getIndex(): string;
```

Comment:
It's me, nitpicking again :) but I would name that `getIdentifier`, as an index with a non-numerical value can be pretty unexpected for an API user.
Of course we are the user, so who cares ^^

Reply:
The only non-numerical value is for the fees, I think it's better to gather everything in the same call instead of having to create 2 functions, for the actual indices and for the fees. we might rename that later though